pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -387,10 +387,6 @@ func (r *Builder) mapDisks(vm *model.VM, persistentVolumeClaims []core.Persisten
 	var kDisks []cnv.Disk
 
 	disks := vm.Disks
-	// TODO might need sort by the disk id (incremental name)
-	/*sort.Slice(disks, func(i, j int) bool {
-		return disks[i].Key < disks[j].Key
-	})*/
```

Comment:
can we also drop AnnImportBackingFile that is used below?

Reply:
the sort is supposed to be (if needed, because i didn't test it) - in cases where we have multiple disks and we get the wrong disk the the one with operation system which may cause to manual changing the disks order/bootable.

Reply:
Its not needed because its already sorted in the ovf file so its being saved like that in the inventory (tested it).
The only thing i'm checking is if the first disk in the ovf is really the bootable one always.

Reply:
@ahadas cool, so just verifying the AnnImportBackingFile change and uploading the changes.